src/HandlerFromPipeSpecFactory.php

```diff
@@ -44,14 +45,16 @@ public function createFromMiddlewareParam(ContainerInterface $container, $middle
             return $this->createPipeFromSpec($container, $middleware);
         }
 
-        throw new InvalidMiddlewareException(sprintf(
-            'Middleware can be one of: string container id, %s, %s, %s or %s; %s given',
-            MiddlewareInterface::class,
-            RequestHandlerInterface::class,
-            Closure::class,
-            PipeSpec::class,
-            is_object($middleware) ? get_class($middleware) : gettype($middleware)
-        ));
+        throw new InvalidMiddlewareException(
+            sprintf(
+                'Middleware can be one of: string container id, %s, %s, %s or %s; %s given',
+                MiddlewareInterface::class,
+                RequestHandlerInterface::class,
+                Closure::class,
+                PipeSpec::class,
+                is_object($middleware) ? get_class($middleware) : gettype($middleware)
+            )
+        );
```

Comment:
Are these changes related to the Psalm analysis?

Reply:
yeah, it's the auto-formatting of the IDE. I haven't really reviewed the code myself, yet. I just opened a draft PR so no one else starts to work one the same thing.
I'll mark the PR as ready for review, when it's finished. okay? 🙂 
I think you can also add the PR to the [2.0.0](https://github.com/laminas/laminas-mvc-middleware/milestone/1) milestone. @Xerkus wants to release 2.0 with psalm integration. Thanks!

Reply:
@heiglandreas 
> yeah, it's the auto-formatting of the IDE.

Please revert these changes otherwise it will be displayed again as an error when switching to version 2 of the coding standard. See: https://docs.laminas.dev/laminas-coding-standard/v2/coding-style-guide/#4-7-method-and-function-calls

> I haven't really reviewed the code myself, yet. I just opened a draft PR so no one else starts to work one the same thing.
I'll mark the PR as ready for review, when it's finished. okay?

My comment only referred to these changes, everything else is okay. 😃

Reply:
thanks for the info! Yes I generally try not to do any style changes on not-style PRs.